spec/peep_spec.rb

```diff
@@ -0,0 +1,26 @@
+describe '.all' do
+  it 'returns a list of all peeps' do
+    connection = PG.connect(dbname: 'chitter_test')
+
+    # Add the test data
+    connection.exec("INSERT INTO peeps (content) VALUES ('Hi there this a test');")
+   
+    peeps = Peep.all
+    peeps = peeps[0].content
+
```

Comment:
again, why are there two peeps? if the first one is being overwritten by the second you could write peeps = Peep.all[0].content